packages/bundler/index.js

```diff
@@ -1,10 +1,10 @@
 const {rollup} = require('rollup');
 const json = require('@rollup/plugin-json');
-const multiInput = require('rollup-plugin-multi-input');
+// const multiInput = require('rollup-plugin-multi-input');
```

Comment:
did you comment these out because you're going to be using them in commit soon, or can they just be deleted 🔥?